practice-two/src/styles/layouts/container.css

```diff
@@ -0,0 +1,18 @@
+.container {
+  margin-left: auto;
+  margin-right: auto;
+  width: 93.75vw;
```

Comment:
Don't set fixed width here, you should use `margin: 0 ...px` to set margin left and right